exercises/allergies/src/example.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _ALLERGIES_H
+#define _ALLERGIES_H
+
+#include <stdbool.h>
+
+typedef enum {
+   Allergen_First = 0,
+   Allergen_Eggs = Allergen_First,
+   Allergen_Peanuts,
+   Allergen_Shellfish,
+   Allergen_Strawberries,
+   Allergen_Tomatoes,
+   Allergen_Chocolate,
+   Allergen_Pollen,
+   Allergen_Cats,
+   Allergen_Last = Allergen_Cats
+} Allergen_t;
+
+typedef struct {
+   int count;
+   Allergen_t *allergens;
+} Allergen_List_t;
+
+bool is_allergic_to(Allergen_t allergen, int score);
```

Comment:
Shouldn't the score be an unsigned value? If you're treating this as a bitfield it will be safer and less confusing to have it be an unsigned.